backend/build.gradle.kts

```diff
@@ -75,3 +76,7 @@ application {
 tasks.withType<JavaExec>().configureEach {
     systemProperties = properties
 }
+
+tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile> {
```

Comment:
Without this I got the following error:
> Cannot inline bytecode built with JVM target 1.8 into bytecode that is being built with JVM target 1.6

Got this from here: https://stackoverflow.com/a/50991772

Is this fine?

Reply:
Thats fine. Its becaude of the jackson yaml module, right?

Reply:
No, because of the `requireObject` method of clickt